src/w3c/level.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * This module updates the title and shortName
+ * of a spec when there is a valid level configuration.
+ *
+ * Levels should be integers >= 0.
+ */
+
+import { getIntlData, showInlineWarning } from "../core/utils.js";
+export const name = "w3c/level";
+
+const localizationStrings = {
+  en: {
+    level: "Level",
+  },
+};
+
+const l10n = getIntlData(localizationStrings);
+
+export function run(conf) {
+  if (!conf.hasOwnProperty("level")) return;
+
+  const h1Elem = document.querySelector("h1#title");
+
+  const level = parseInt(conf.level);
+  if (!Number.isInteger(level) || level < 0) {
+    showInlineWarning(
```

Comment:
As this is a MUST, we probably want a proper error here. 

```suggestion
    showInlineError(
```